testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/oid4vc/issuance/signing/OID4VCSdJwtIssuingEndpointTest.java

```diff
@@ -391,7 +398,8 @@ protected void handleCredentialResponse(CredentialResponse credentialResponse) t
             assertTrue("The credentials should include the email claim.", disclosureMap.containsKey("email"));
             assertEquals("email claim incorrectly mapped.", disclosureMap.get("email").get(2).asText(), "john@email.cz");
 
-        }
+            assertNotNull("Test credential shall include an iat claim.", jsonWebToken.getIat());
```

Comment:
I think that It might be better to check the value of `jsonWebToken.getIat()` in a reasonable range. WDYT?

Reply:
This is not the purpose of the test. IAT is a value specific to the credential. It might be way off if e.g. issuer is leveling the iat to the first second of the year for privacy purpose.